graphbolt/src/cuda/extension/unique_and_compact.h

```diff
@@ -0,0 +1,26 @@
+/**
+ *  Copyright (c) 2023, GT-TDAlab (Muhammed Fatih Balin & Umit V. Catalyurek)
+ * @file cuda/unique_and_compact.h
+ * @brief Unique and compact operator utilities on CUDA using hash table.
+ */
+
+#ifndef GRAPHBOLT_CUDA_UNIQUE_AND_COMPACT_H_
+#define GRAPHBOLT_CUDA_UNIQUE_AND_COMPACT_H_
+
+#include <torch/script.h>
+
+#include <vector>
+
+namespace graphbolt {
+namespace ops {
+
+std::vector<std::tuple<torch::Tensor, torch::Tensor, torch::Tensor> >
```

Comment:
typo: extra space

Reply:
The formatter automatically does that. Can't fix it.